### PR TITLE
Styling the checkbox list

### PR DESCRIPTION
**Pull request by Markvanderuit**
Replaces CheckboxComboCheckbox in the DirectorShotDetailView with a MenuButton with styling. Class is rather volatile due to some javaFX issues concerning who/what starts running first, but in this specific case it works just fine.

Also adds checkboxes to Creation modal views, so instruments can be selected more easily.

### CODE REVIEW

File: src/main/java/control/DetailViewController.java [hunk 1]
```diff
@@ -102,7 +115,8 @@ private void beforePaddingUpdateHelper() {
             DirectorShot directorShot = ((DirectorShot) manager.getActiveShotBlock().getShot());
             
             directorShot.setFrontShotPadding(newVal);
-            directorShot.getCameraShots().forEach(e -> {
+            directorShot.getCameraShots().forEach(
+                e -> {
```
Comment (martijnjanssen) [inline src/main/java/control/DetailViewController.java]: You can just leave this one on the line above. This adds an unnecessary line in your method. It does not make the method more readable, it does not make it worse either.

**Comment by martijnjanssen** [comment]
Looks good!

**Comment by Markvanderuit** [comment]
Closed for now due to unforeseen test issues.

File: src/main/java/control/DetailViewController.java [hunk 2]
```diff
@@ -1,17 +1,26 @@
 package control;
 
-import java.util.Iterator;
-import java.util.Set;
```
Comment (bart-de-jonge) [inline src/main/java/control/DetailViewController.java]: whitespace

File: src/main/java/control/DetailViewController.java [hunk 3]
```diff
@@ -405,34 +407,59 @@ public void activeBlockChanged() {
...
-        ((DirectorDetailView) detailView).getSelectCamerasDropDown().getItems().clear();
-        manager.getScriptingProject().getCameras().forEach(camera -> {
-                ((DirectorDetailView) detailView).getSelectCamerasDropDown()
-                    .getItems().add(camera.getName());
-            });
-        indices.forEach(e -> {
-                ((DirectorDetailView) detailView).getSelectCamerasDropDown()
-                    .getCheckModel().check(e);
-            });
+    private void initDropDown() {
```
Comment (bart-de-jonge) [inline src/main/java/control/DetailViewController.java:426]: Pretty long method with nested loops and ifs. Maybe think about splitting it or adding some comments

**Comment by bart-de-jonge** [comment]
2 small comments, please take a look

File: src/main/java/control/DetailViewController.java (hunk 3 above)
Comment (Markvanderuit) [inline src/main/java/control/DetailViewController.java:426]: Split up into three methods.

File: src/main/java/gui/headerarea/DirectorDetailView.java [hunk 4]
```diff
@@ -78,17 +84,15 @@ private void initPaddingAfter() {
         paddingAfterBox.setAlignment(Pos.CENTER);
         directorItemsBox.getChildren().add(paddingAfterBox);
     }
-    
+
     /**
-     * Init the dropdown menu to select cameras.
+     * Init experimental dropdown menu to select cameras.
```
Comment (martijnjanssen) [inline src/main/java/gui/headerarea/DirectorDetailView.java:89]: :scream:

**Comment by martijnjanssen** [comment]
Looks good!

**Pull request merged**